### DISCUSSION TITLE
java.lang.IllegalStateException: Software rendering doesn't support hardware bitmaps

### DISCUSSION THREAD

**Discussion by mrosasbr** [PatilShreyas/Capturable#180] [Q&A] [answered]
I am experiencing this error on Android Oreo devices. Is there any known solution for this issue?

**Comment by PatilShreyas** [comment, marked as answer]
Could you let me know if you are using the image loader library? If yes, so wherever you're capturing the content, you'll need to disable the hardware bitmap.
While capturing the content on the devices with Android OS version O and above (API 26+) and network images like Coil, Picasso, Glide, etc., it may throw errors like java.lang.IllegalArgumentException: Software rendering doesn't support hardware bitmaps

**Reply by mrosasbr** [reply]
I'm capturing external images loaded with Coil. Adding the following resolved the error: .allowHardware(false). Thank you for the response!
